index.css

```diff
@@ -0,0 +1,3 @@
+.header-title {
```

Comment:
- Kindly fix your linter issues displayed on your Pull Request.

Reply:
Hello @carlos-ssh,

Thank you for the review.

I have run linter tests locally and I see no errors raised. I am currently running node 14.17.6 and post taking a read on some similar solutions to the error, online resources are advising that I delete the node modules and package.lock.json.

What would you advise that I do in this scenario?

Reply:
It's needed to check locally and in your pull request which error do you have.  you can go to your linters described in the next image and get a detail about your issue. Solve it and after that request another code review.

<img width="1440" alt="Screen Shot 2021-09-14 at 7 56 00" src="https://user-images.githubusercontent.com/22258274/133261499-051a8832-a882-409f-9e52-7efa9c491d24.png">

Reply:
Hello, once more @carlos-ssh,

Thank you a lot for that insightful suggestion.

I have taken some time to look at the error raised and it seems it's coming from a missing dependency program.

Kindly see the attached herewith image for your reference. Meanwhile, I will go ahead and make another pull request as you advised earlier.

![stylelint error](https://user-images.githubusercontent.com/35315311/133266621-33811776-dfb1-4377-a925-c08a6f92155d.png)

Reply:
Hi @carlos-ssh,

>I believe I am done following your advice.

I am requesting another review, and hopefully this time I check out fine 😃